### PR TITLE
RNA-seq workshop in December

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by bgruening** [comment]
@bebatut ready for review.

**Comment by erasche** [comment]
LGTM!

File: _events/2018-09-13-Announcement-Galaxy-course.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+<!-- TODO: map -->
+
+> Note:
+> In Freiburg you sometimes have to pay an additional accommodation tax. For business trips this tax does not
+> to be paid if your employer fills out [this form](http://www.freiburg.de/servicebw/UebernachtungSt_Arbeitgeberbescheinigung.pdf). You will need to show this form to the hotel.
+{:.alert.alert-warning}
+
+## Organizers
+
+{% assign extra_organizers =  "galaxy-freiburg|galaxy-europe|mpi" | split: "|"  %}
```
Comment (bebatut) [inline _events/2018-09-13-Announcement-Galaxy-course.md]: MPI as organizer?

File: _events/2018-09-13-Announcement-Galaxy-course.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+accommodation costs.
+
+## Venue
+
+{% include map.html location=page.location showmap=true zoomlevel=15 hidepopup=true %}
+
+
+# Important notes
+
+1. If you are registered but **cannot attend** our workshop, please [contact us
```
Comment (bebatut) [inline _events/2018-09-13-Announcement-Galaxy-course.md]: Maybe putting that after the registration

File: _events/2018-09-13-Announcement-Galaxy-course.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+   by email](mailto:contact@usegalaxy.eu) immediately. We have a long waiting
+   list and can give your place to others, even on short notice.
+2. Please register at [https://usegalaxy.eu](https://usegalaxy.eu) if you have
```
Comment (bebatut) [inline _events/2018-09-13-Announcement-Galaxy-course.md]: Is it valid for this workshop? Given the information before

**Comment by bgruening** [comment]
@bebatut better?

**Pull request merged**